mex/common/models/base/extracted_data.py

```diff
@@ -96,3 +96,7 @@ def _get_stable_target_id(
             .assign(self.hadPrimarySource, self.identifierInPrimarySource)
             .stableTargetId
         )
+
+    def __hash__(self) -> int:
```

Comment:
this change should also go into the changelog

Reply:
Done